automation/build-artifacts.repos.el7

```diff
@@ -1,2 +1,3 @@
 http://jenkins.ovirt.org/job/ovirt-engine-nodejs_master_build-artifacts-el7-x86_64/lastSuccessfulBuild/artifact/exported-artifacts/
 http://jenkins.ovirt.org/job/ovirt-engine-nodejs-modules_master_build-artifacts-el7-x86_64/lastSuccessfulBuild/artifact/exported-artifacts/
+http://jenkins.ovirt.org/job/ovirt-engine-yarn_master_build-artifacts-el7-x86_64/lastSuccessfulBuild/artifact/exported-artifacts/
```

Comment:
Yes it should, but it can't. This is actually a `yum` issue. When you issue a `yum` command like this:

    yum install -y foo bar baz

And the `bar` package is not found, `yum` unfortunately does not return a failure result error. This makes `mock` carry on happily. Suggestions for workarounds that will not prevent the usage of `mock`'s cache are welcome.